src/components/LoanCollections/KivaClassicLoanCarousel.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+	<div class="tw-pt-4">
+		<transition name="kvfade">
+			<div
+				v-if="isLoading"
+				class="spinner"
+			>
+				<kv-loading-spinner />
+			</div>
+		</transition>
+
+		<kv-carousel
+			v-if="augmentedLoanIds.length > 0 && isVisible"
+			class="tw-w-full tw-overflow-visible md:tw-overflow-hidden"
+			:autoplay="false"
+			:embla-options="{
+				loop: false,
+				align: 'start'
+			}"
+			indicator-style="none"
+			:multiple-slides-visible="true"
+			slides-to-scroll="visible"
+			:slide-max-width="singleSlideWidth"
+			@interact-carousel="onInteractCarousel"
+		>
+			<template v-for="(loanId, index) in augmentedLoanIds" #[`slide${index}`]>
+				<!-- Show View more Card -->
+				<router-link
+					v-if="loanId === 3"
+					:key="`view-more-card-${loanId}`"
+					class="tw-flex tw-items-center tw-h-full tw-w-full hover:tw-bg-brand-50 tw-rounded"
```

Comment:
Since `tw-bg-brand-50` is a static color, it will always be light-green no matter what theme. So in a future where dark mode applies the dark theme, this hover state will have light-green background and white text.

There has been some talk about a `tw-bg-accent` color (that might be light green in default theme), but it doesn't currently exist.

I'd suggest either `hover:tw-bg-secondary` or `hover:tw-bg-action-highlight hover:tw-text-primary-inverse` instead of the brand-50.